create-schemas/schemaconf.json

```diff
@@ -48,7 +48,7 @@
     "release": "v1.6.4",
     "ramlPath": "src/main/resources/swagger.api",
     "copyFiles": [
-      "etc/mod-search-instances.raml",
+      "create-schemas/etc/mod-search-instances.raml",
```

Comment:
@MikeTaylor - From my PR description it might not be clear that without this change (for me at least) create-schemas.js throws a file not found type of error.